src/vs/workbench/contrib/preferences/electron-browser/settingsEditor2.ts

```diff
@@ -1077,6 +1080,7 @@ export class SettingsEditor2 extends BaseEditor {
 			}
 
 			this.refreshTOCTree();
+			this.tocTree.expandAll();
```

Comment:
The reason that `this.tocTree.expandAll()` is added here is the first time the settings editor being opened with predefined query, the TOC is collapsed even though the tree item `collapsed` state is `undefined`.

Reply:
Won't this be covered by the expandAll call above on 1071?

Reply:
Removed.